README.md

```diff
@@ -1,11 +1,294 @@
-# Thanx SDK for iOS
-
-Thanx SDK provides a simple way to get the Thanx customer engagement experience into your mobile app.
-
-## Documentation
-
-Pending
-
-## Demo
-
-Pending
+# Thanx iOS SDK
+
+The Thanx iOS SDK provides a simple way to embed the Thanx customer engagement
+experience in iOS applications.
+
+## Delivery
+
+The SDK will be delivered as a native Swift library distributed via the package
+manager Cocoapods.
+
+## Authentication
+
+### Merchant
+
+Thanx will provide the SDK `Client ID` and a `Client Secret` via a secure
+channel.
```

Comment:
would it be worth adding this to the dashboard somewhere?

Reply:
For now, we'll be exchanging credentials with them via SFTP